todo.js

```diff
@@ -15,3 +15,39 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+const fs = require('fs');
+let todoList = fs.readFileSync("todos.txt","utf-8");
+let command = process.argv[2];
+let readLineSync = require("readLine-sync");
+let TODO = {};
+function list() {
+
+  if (command === "list")
+  {
+    console.log(todoList);
+  }
+}
+list();
+function append() {
+  if (command === "append")
+  {
+    let newTask = readLineSync.question("What would you like to add?");
+    console.log(todoList + newTask);
+  }
+}
+append();
+function delete() {
+  if (command === "delete")
+  {
+    let deleteTask = readLineSync.question("What would you like to delete?");
+    if (todoList.includes(deleteTask)) {
+      for (let i =0;i<todoList.length;i++) {
+        if (todoList[i] === deleteTask && todoList[i+1] === deleteTask[i+1]) {
+          todoList.splice(todoList[i],deleteTask.length);
```

Comment:
`todoList` is a string and `splice` doesn't exist for a string. You seem to be conceptualizing it as an array, so don't understand why you don't actually make it an array.

If you're imagining `todoList` is an array of strings then `todoList[i]` is a string (the task description, probably).

If you're imagining `todoList` is a string then `todoList[i]` is a single character.

Either way, the first argument to `splice` is a number, so passing in `todoList[i]` as an argument makes no sense.

See https://github.com/PMcS3/javascript-milestones/blob/arrays/array.js#L33 for an example of how `splice` works.